Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -25,7 +25,7 @@ final class PledgePaymentMethodsViewController: UIViewController {
   private lazy var cardsStackView: UIStackView = { UIStackView(frame: .zero) }()
   internal weak var delegate: PledgePaymentMethodsViewControllerDelegate?
   internal weak var messageDisplayingDelegate: PledgeViewControllerMessageDisplaying?
-  private lazy var pledgeButton: UIButton = { UIButton.init(type: .custom) }()
+  private(set) lazy var pledgeButton: LoadingButton = { LoadingButton(frame: .zero) }()
```

Comment:
I can expose this another way but thought this was simplest for what we need to do (RAC binding)